site/layouts/_default/reference-index.html

```diff
@@ -50,6 +50,22 @@
           {{ end }}
           </ul>
           {{ end }}
+          <h1 id="how-do-i-use">How do I use this documentation?</h1>
+          <p>Each specification consists of an outline, an index, and a reference, for both JSON and XML formats.</p>
+          <h2 id="outline-pages">Outline Pages</h2>
+          <p>An outline is a navigatable, tree-based representation of OSCAL's models. For example, below is the collapsed OSCAL System Security Plan outline in JSON and XML, respectively. The name links to the corresponding entry in the XML or JSON formats.</p>
+          <p>By clicking on the black triangle in front of an entry, it expands to show child entries.
+          </p>
+          <img src="/img/outline-ssp-json.png"/>
+          <img src="/img/outline-ssp-xml.png"/>
+          <p>In an outline, the cardinality, the minimum and maximum number of instances required, follows the entry as a number in square brackets. The data type follows the cardinality (see the cardinality <code>[1]</code> in the image below).</p>
+          <h2 id="reference-pages">Reference Pages</h2>
+          <p>In an outline, each entry links to its corresponding reference providing a definition for an entry and its constraints.</p>
+          <img src="/img/reference-ssp-json.png"/>
+          <h2 id="index-pages">Index Pages</h2>
+          <p>An index for each OSCAL model lists all entries in alphabetical order.</p>
+          <p>Each entry in the index lists a specific location in the applicable model, in both JSON and XML formats.</p>
+          <img src="/img/index-ssp-props-json.png"/>
```

Comment:
Concern: I am not sure this approach is maybe hacky and I am not sure how to make a special block to put in `content/models/_index.md` correctly while keeping the release number listing "in the middle" and not make expert users strain for novices or vice versa. Would love feedback.

Reply:
Nikita actually mentioned an interesting workaround for this that I will try with a file for additional content that is a trailer to the list, it is less kludgey than this.

Reply:
OK I finally figured out a way to do it in a way more conducive to maintenance.